bundles/org.openhab.binding.deutschebahn/README.md

```diff
@@ -0,0 +1,345 @@
+                    style:
+                      color: "=items[props.changed_time].state != 'NULL' && items[props.changed_time].state != items[props.planned_time].state ? 'red' : ''"
+          - component: f7-col
+            config:
+              width: 75
+            slots:
+              default:
+                - component: Label
+                  config:
+                    text: "=(items[props.changed_state].state == 'c' ? 'Zug fällt aus - ' : '') + (items[props.messages].state != 'NULL' ? items[props.messages].state : '')"
+                    style:
+                      color: red
+          - component: f7-col
+            config:
+              width: 10
+            slots:
+              default:
+                - component: Label
+                  config:
+                    text: "=items[props.changed_platform].state != 'NULL' ? items[props.changed_platform].state :  items[props.planned_platform].state"
+                    style:
+                      color: "=items[props.changed_platform].state != 'NULL' ? 'red' : ''"
+                      text-align: right
+    - component: f7-row
+      slots:
+        default:
+          - component: f7-col
+            config:
+              width: 15
+            slots:
+              default:
+                - component: Label
+                  config:
+                    text: "=items[props.line].state != 'NULL' ? (items[props.category].state + ' ' + items[props.line].state) : (items[props.category].state + ' ' + items[props.number].state)"
+          - component: f7-col
+            config:
+              width: 50
+            slots:
+              default:
+                - component: Label
+                  config:
+                    text: "=items[props.changed_intermediate_stations].state != 'NULL' ? items[props.changed_intermediate_stations].state : items[props.planned_intermediate_stations].state"
+                    style:
+                      color: "=items[props.changed_intermediate_stations].state != 'NULL' ? 'red' : ''"
+          - component: f7-col
+            config:
+              width: 35
+            slots:
+              default:
+                - component: Label
+                  config:
+                    text: "=items[props.changed_final_station].state != 'NULL' ? items[props.changed_final_station].state : items[props.planned_final_station].state"
+                    style:
+                      color: "=items[props.changed_final_station].state != 'NULL' ? 'red' : ''"
+                      font-weight: bold
+                      text-align: right
+```
+
+
+Using the widget for displaying the next four departures:
+
+![Departures Hannover HBF](doc/images/Abfahrten_HannoverHBF.png "openHAB page with four widgets displaying the next departures at Hannover HBF")
```

Comment:
I think images need to reside directly under `doc/` to be rendered correctly on the OH webpage.

Reply:
fixed